src/doc/reference/src/visibility-and-privacy.md

```diff
@@ -40,7 +40,10 @@ With the notion of an item being either public or private, Rust allows item
 accesses in two cases:
 
 1. If an item is public, then it can be used externally through any of its
-   public ancestors.
+   accessible ancestors (either by anyone that has an access to an ancestor
+   in the case of a chain of public ancestors,
+   or by the siblings of a private ancestor, in the case of a chain of public
+   ancestors terminating into a private ancestor).
```

Comment:
I might have expressed the consequences unclearly, but my point that the paragraphs 1 and 2 don't cover every case stands. The paragraph 1 incorrectly suggests that a public item is only usable through public ancestors, but it is usable through *nameable* (either public, or private but local) ancestors.